cypress/e2e/4-node-creator.cy.ts

```diff
@@ -1,16 +1,14 @@
-import { DEFAULT_USER_EMAIL, DEFAULT_USER_PASSWORD } from "../constants";
-import { randFirstName, randLastName } from "@ngneat/falso";
 import { NodeCreator } from '../pages/features/node-creator';
 import { INodeTypeDescription } from '../../packages/workflow';
 import CustomNodeFixture from '../fixtures/Custom_node.json';
 
-const username = DEFAULT_USER_EMAIL;
-const password = DEFAULT_USER_PASSWORD;
-const firstName = randFirstName();
-const lastName = randLastName();
 const nodeCreatorFeature = new NodeCreator();
 
 describe('Node Creator', () => {
+	before(() => {
+		cy.task('db:reset');
+	})
```

Comment:
Why is this needed? None of the others reset in `before`, plus `beforeEach` should cover it?

Reply:
The other `cy.task` was unnecessary, forgot to delete it.